pkg/controller/installation/installer_test.go

```diff
@@ -303,3 +304,191 @@ func TestInstallerBrokenChartContents(t *testing.T) {
 		t.Fatal("installRelease should fail, invalid k8s objects")
 	}
 }
+
+func TestInstallerSingleServiceNoLB(t *testing.T) {
```

Comment:
I think it might make sense to check the contents of the created Services, so we can check that they have the Selectors that we expect (unmodified if a secondary Service and the primary has the 'shipper-lb' label; modified if its the only Service or it has the shipper-lb prod label, etc.)

Reply:
We do check the contents of Service objects in validateServiceCreateObject and in validateDeploymentCreateObject + this commit enforces a precise label check: https://github.com/bookingcom/shipper/pull/43/files#diff-2e47c909741e399c1373620e5e4f379aL170